app/src/main/java/com/asfoundation/wallet/util/StoreOemAddresses.kt

```diff
@@ -0,0 +1,3 @@
+package com.asfoundation.wallet.util
+
```

Comment:
We don't need to create This class in a different file, since the context in which is used is only the LocalPaymentInteractor, please add it to the LocalPaymentInteracto file.